spec/requests/api/v1/serializer_options_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'cgi'
+require 'uri'
+
+describe 'Serializer options', type: :request do
+  it 'adds meta with pagination information' do # rubocop:disable RSpec/MultipleExpectations
+    get '/api/v1/locations', params: {}, headers: default_headers
+
+    expect(response).to have_http_status(:ok)
+    expect(json_response).to have_key('meta')
+    meta = json_response['meta']
+    expect(meta).to have_key('current_page')
```

Comment:
Maybe `expect(meta.keys).to include('current_page', ...)`? I'm not sure how the error output is with have_key tho, I haven't tried it before 😓

Reply:
🤘 looks good on my end
```
||   1) Serializer options adds meta with pagination information
||      Failure/Error:
||        expect(json_response['meta'].keys).to include(
||          'current_page', 'total_pages', 'total_count', 'nekaj'
||        )
||      
||        expected ["current_page", "total_pages", "total_count", "padding", "page_size", "max_page_size"] to include "nekaj"
```